0.10/README.md

```diff
@@ -87,14 +87,15 @@ Environment variables
 
 Application developers can use the following environment variables to configure the runtime behavior of this image:
 
-NAME        | Description
-------------|-------------
-NODE_ENV    | NodeJS runtime mode (default: "production")
-DEV_MODE    | When set to "true", `nodemon` will be used to automatically reload the server while you work (default: "false"). Setting `DEV_MODE` to "true" will change the `NODE_ENV` default to "development" (if not explicitly set).
-NPM_RUN     | Select an alternate / custom runtime mode, defined in your `package.json` file's [`scripts`](https://docs.npmjs.com/misc/scripts) section (default: npm run "start"). These user-defined run-scripts are unavailable while `DEV_MODE` is in use.
-HTTP_PROXY  | use an npm proxy during assembly
-HTTPS_PROXY | use an npm proxy during assembly
-NPM_MIRROR  | use a custom NPM registry mirror to download packages during the build process
+NAME          | Description
+--------------|-------------
+NODE_ENV      | NodeJS runtime mode (default: "production")
+DEV_MODE      | When set to "true", `nodemon` will be used to automatically reload the server while you work (default: "false"). Setting `DEV_MODE` to "true" will change the `NODE_ENV` default to "development" (if not explicitly set).
+NPM_RUN       | Select an alternate / custom runtime mode, defined in your `package.json` file's [`scripts`](https://docs.npmjs.com/misc/scripts) section (default: npm run "start"). These user-defined run-scripts are unavailable while `DEV_MODE` is in use.
+HTTP_PROXY    | use an npm proxy during assembly
+HTTPS_PROXY   | use an npm proxy during assembly
+NPM_MIRROR    | use a custom NPM registry mirror to download packages during the build process
+NPM_LOG_LEVEL | specify the logging output for npm. Options ("info","warn","silent"). Default to warn.
```

Comment:
The options shouldn't be this restrictive, we actually allow all values that npm understands `npm help 7 config` says silent, warn, info, verbose and silly.
I would document this by simply saying "any value that npm understands, e.g., silent, warn, info, verbose and silly"